scripts/search.js

```diff
@@ -29,6 +29,18 @@
             $searchInput.focus();
         });
 
+        // Open the search when pressing / or ?
+        // Use 'keypress' events to handle key combos
+        $(document).on('keypress', function(event) {
+            if (event.keyCode !== 47 && event.keyCode !== 63) {
+                return;
+            }
+
+            event.preventDefault();
+            $toggleCheckbox.click();
+            $searchInput.focus();
```

Comment:
I noticed that we can't use `/` or `?` in search input anymore since it will click / focus it.

If the user is currently typing, we need to ignore these events (exept for espace).

What do you think about this ?